File: benchmarks/Java/cf_study/loop_fis.java
public void foo(int[] arr) {
    for (int i=1; i < arr.length; i++) {
        System.out.println(arr[i]);
    }
}

public void foo(int[] arr) {
    for (int i=1; i < arr.length/2; i++) {
        System.out.println(arr[i]);
    }
    for (int i=arr.length/2; i < arr.length; i++) {
        System.out.println(arr[i]);
    }
}

File: benchmarks/C/cf_study/loop_fis.c
/* void foo(int[] arr, int len) { */
/*   for (int i=1; i < len; i++) { */
/*     printf("%d\n", arr[i]); */
/*   } */
/* } */

void foo(int[] arr, int len) {
  for (int i=1; i < len/2; i++) {
    printf("%d\n", arr[i]);
  }
  for (int i=len/2; i < len; i++) {
    printf("%d\n", arr[i]);
  }
}
